sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v2/query.sql

```diff
@@ -0,0 +1,291 @@
+WITH events_unnested AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    category AS event_category,
+    name AS event_name,
+    timestamp AS event_timestamp,
+    client_info,
+    metadata,
+    normalized_os,
+    normalized_os_version,
+    normalized_country_code,
+    normalized_channel,
+    ping_info,
+    extra AS event_details,
+    metrics
+  FROM
+    -- https://dictionary.telemetry.mozilla.org/apps/firefox_desktop/pings/newtab
+    `moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1`,
+    UNNEST(events)
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+    AND category IN ('newtab', 'topsites', 'newtab.search', 'newtab.search.ad', 'pocket')
+    AND name IN ('closed', 'opened', 'impression', 'issued', 'click', 'save', 'topic_click')
+),
+visit_metadata AS (
+  SELECT
+    mozfun.map.get_key(event_details, "newtab_visit_id") AS newtab_visit_id,
+    submission_date,
+    ANY_VALUE(client_info.client_id) AS client_id,
+    ANY_VALUE(metrics.uuid.legacy_telemetry_client_id) AS legacy_telemetry_client_id,
+    ANY_VALUE(normalized_os) AS normalized_os,
```

Comment:
Are all of these `ANY_VALUE` fields _guaranteed_ to be the same for every event from a single `newtab_visit_id` (e.g. is a single ping tied to a single `newtab_visit_id`)? If not, we may end up with different values on reruns/backfills (i.e. not idempotent)